### PR TITLE
Rename create_replica_method to create_replica_methods

### PR DESCRIPTION
**Pull request by erthalion**
To make it clear that it's actually an array. But of course for backward compatibility accept both.

### CODE REVIEW

File: docs/SETTINGS.rst [hunk 1]
```diff
@@ -131,7 +131,7 @@ PostgreSQL
 -  **pg\_ctl\_timeout**: How long should pg_ctl wait when doing ``start``, ``stop`` or ``restart``. Default value is 60 seconds.
 -  **use\_pg\_rewind**: try to use pg\_rewind on the former leader when it joins cluster as a replica.
 -  **remove\_data\_directory\_on\_rewind\_failure**: If this option is enabled, Patroni will remove postgres data directory and recreate replica. Otherwise it will try to follow the new leader. Default value is **false**.
--  **replica\_method**: for each create_replica_method other than basebackup, you would add a configuration section of the same name. At a minimum, this should include "command" with a full path to the actual script to be executed. Other configuration parameters will be passed along to the script in the form "parameter=value".
+-  **replica\_method**: for each create_replica_methods other than basebackup, you would add a configuration section of the same name. At a minimum, this should include "command" with a full path to the actual script to be executed. Other configuration parameters will be passed along to the script in the form "parameter=value".
```
Comment (Firefishy) [inline docs/SETTINGS.rst]: The key here should likely be changed to the new plural form.

Comment (erthalion) [reply docs/SETTINGS.rst]: Good catch, thanks

**Comment by CyberDem0n** [comment]
:+1:

**Comment by erthalion** [comment]
:+1:

**Pull request merged**